<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <widget class="QMenu" name="menu_New">
     <property name="title">
      <string>&amp;New</string>
     </property>
     <addaction name="action_Project"/>
    </widget>
    <addaction name="menu_New"/>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Save"/>
    <addaction name="action_Save_All"/>
    <addaction name="action_Save_As"/>
    <addaction name="separator"/>
    <addaction name="action_Print"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <widget class="QMenu" name="menu_Edit">
    <property name="title">
     <string>&amp;Edit</string>
    </property>
    <widget class="QMenu" name="menu_Search_n_Replace">
     <property name="title">
      <string>&amp;Search 'n Replace</string>
     </property>
     <addaction name="actionSearch_n_Replace"/>
     <addaction name="actionSearch_n_Replace_in_Files"/>
    </widget>
    <addaction name="action_Undo"/>
    <addaction name="action_Redo"/>
    <addaction name="separator"/>
    <addaction name="action_Cut"/>
    <addaction name="actionC_opy"/>
    <addaction name="action_Paste"/>
    <addaction name="separator"/>
    <addaction name="action_Select_All"/>
    <addaction name="separator"/>
    <addaction name="menu_Search_n_Replace"/>
    <addaction name="separator"/>
    <addaction name="action_Go_to"/>
   </widget>
   <widget class="QMenu" name="menu_Project">
    <property name="title">
     <string>&amp;Project</string>
    </property>
    <addaction name="actionAdd_item"/>
    <addaction name="actionRemove_Item"/>
    <addaction name="separator"/>
    <addaction name="actionProperties"/>
   </widget>
   <widget class="QMenu" name="menu_Debug">
    <property name="title">
     <string>&amp;Debug</string>
    </property>
    <addaction name="actionExecute"/>
    <addaction name="separator"/>
    <addaction name="actionStart"/>
    <addaction name="actionStop"/>
    <addaction name="actionPause"/>
    <addaction name="separator"/>
    <addaction name="action_Step_in"/>
    <addaction name="actionStep_out"/>
    <addaction name="actionStep_into"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="action_Contents"/>
    <addaction name="separator"/>
    <addaction name="action_About"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Edit"/>
   <addaction name="menu_Project"/>
   <addaction name="menu_Debug"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_Save_All">
   <property name="text">
    <string>&amp;Save All</string>
   </property>
  </action>
  <action name="action_Save_As">
   <property name="text">
    <string>&amp;Save As ...</string>
   </property>
  </action>
  <action name="action_Print">
   <property name="text">
    <string>&amp;Print ...</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="action_Undo">
   <property name="text">
    <string>&amp;Undo</string>
   </property>
  </action>
  <action name="action_Redo">
   <property name="text">
    <string>&amp;Redo</string>
   </property>
  </action>
  <action name="action_Cut">
   <property name="text">
    <string>&amp;Cut</string>
   </property>
  </action>
  <action name="actionC_opy">
   <property name="text">
    <string>C&amp;opy</string>
   </property>
  </action>
  <action name="action_Paste">
   <property name="text">
    <string>&amp;Paste</string>
   </property>
  </action>
  <action name="action_Select_All">
   <property name="text">
    <string>&amp;Select All</string>
   </property>
  </action>
  <action name="actionExecute">
   <property name="text">
    <string>Execute</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="action_Go_to">
   <property name="text">
    <string>&amp;Go to Line...</string>
   </property>
  </action>
  <action name="action_Step_in">
   <property name="text">
    <string>&amp;Step in</string>
   </property>
   <property name="shortcut">
    <string>F11</string>
   </property>
  </action>
  <action name="actionStep_out">
   <property name="text">
    <string>Step out</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </action>
  <action name="actionStep_into">
   <property name="text">
    <string>Step into</string>
   </property>
   <property name="shortcut">
    <string>F10</string>
   </property>
  </action>
  <action name="action_Project">
   <property name="text">
    <string>&amp;Project ...</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="text">
    <string>Properties ...</string>
   </property>
  </action>
  <action name="action_Contents">
   <property name="text">
    <string>&amp;Contents</string>
   </property>
  </action>
  <action name="action_About">
   <property name="text">
    <string>&amp;About ...</string>
   </property>
  </action>
  <action name="actionSearch_n_Replace">
   <property name="text">
    <string>Search 'n Replace</string>
   </property>
  </action>
  <action name="actionSearch_n_Replace_in_Files">
   <property name="text">
    <string>Search 'n Replace in Files</string>
   </property>
  </action>
  <action name="actionAdd_item">
   <property name="text">
    <string>Add Item ...</string>
   </property>
  </action>
  <action name="actionRemove_Item">
   <property name="text">
    <string>Remove Item ...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
